eth/executionclient/constants.go

```diff
@@ -0,0 +1,10 @@
+package executionclient
+
+const (
+	SlotsPerEpoch    = 32
```

Comment:
this is read from the beacon config and might be different on different networks.. (devnet or some testnet). usually its 32 but it's best to just read all this info from the beacon node. (we have PRs for that and it will probably be merged before this PR.

Reply:
Great, what about `ForkEpoch`? Do we have a chain config or something that can be used here? To be honest, I would love to have something centralized, like `ChainConfig` in geth so that we can maintain everything (forks, sips, checks, e.g., IsFinalityFork(), etc.) in one place

Reply:
we have networkconfig package and a struct that initialized in the main endpoint of the program, it should have all chain related config.

Reply:
@kchojn yes, the list of forks with their epoch will be there too. You can check what it will look like in https://github.com/ssvlabs/ssv/pull/2183

Reply:
Oh, you actually meant SSV fork, not ETH fork, right? We'll need to create everything SSV-related inside the SSV config

Reply:
@nkryuchkov  Yes, SSV fork. I think i will create something like ssv_fork +  will move logic in there with forks (alan and upcoming)

Reply:
Sounds good